metricbeat/module/logstash/node_stats/node_stats_integration_test.go

```diff
@@ -25,28 +25,15 @@ import (
 	"github.com/elastic/beats/libbeat/tests/compose"
 	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
 	"github.com/elastic/beats/metricbeat/module/logstash"
-
-	"github.com/stretchr/testify/assert"
 )
 
-func TestFetch(t *testing.T) {
```

Comment:
I assume this is removed because of the reporter change. But I was not sure where this is covered now?

Reply:
Yes, I removed this because `Fetch` no longer returns an event. And now this file is consistent  with other Elastic stack integration test files; they also only have a `TestData` function that ultimately exercises `Fetch` via the `mbtest.WriteEventsReporterV2` call in the test code.

Reply:
TestData is not run by default in our tests. What I did for the Elasticsearch module you can find here: https://github.com/elastic/beats/blob/master/metricbeat/module/elasticsearch/elasticsearch_integration_test.go It basically imports all the metricsets with a fetcher and then runs the tests automatically. We should this here too for all the ones where `TestFetch` was removed.